main.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, render_template, request, redirect, url_for
+import pandas as pd
+import json
+import plotly
+import plotly.express as px
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+ 
+app = Flask(__name__)
+
+df = pd.read_csv('women_EU2.csv', sep=',')
```

Comment:
We're declaring `df` variable 3 times in this file. Since we need loading it only once, maybe it would make sense to declare once and reuse? See how such a variable (a constant, to be precise) is declared here https://github.com/braintree/braintree_flask_example/blob/main/app.py#L14